### PR TITLE
Update opens

### PR DESCRIPTION
**Pull request by local-minimum**
Part of #39 

Manually tried out how commands seem to work now as it is quite hard to actually test most of these changes at the moment.

### CODE REVIEW

**Review by e-larsson — APPROVED** [review]
LGTM

File: scanomatic/data_processing/pheno/save.py [hunk 1]
```diff
@@ -236,7 +236,7 @@ def zipit(save_functions, data, zip_paths):
     zip_paths.append(
         os.path.join(dir_path, _paths.phenotypes_filter_undo),
     )
-    save_functions.append(lambda x, y: pickle.dump(y, x))
+    save_functions.append(lambda fh, obj: pickle.dump(obj, fh))
```
Comment (skymandr) [inline scanomatic/data_processing/pheno/save.py:239]: Thanks!

File: scanomatic/io/pickler.py [hunk 2]
```diff
@@ -45,7 +45,7 @@ def __init__(self):
         self._next = None
         self.io = None
 
-    def __call__(self, line: str):
+    def __call__(self, line: bytes):
```
Comment (skymandr) [inline scanomatic/io/pickler.py:48]: This file makes me nervous by proxy! Is it still useful, even after moving to json configs?

**Review by skymandr — APPROVED** [review]
lgtm

File: scanomatic/io/pickler.py (hunk 2 above)
Comment (local-minimum) [reply scanomatic/io/pickler.py:48]: unfortunately this is used for saving things that are not `Model`s... so yes this is a very hackish thing that should be removed some day hopefully.

Comment (skymandr) [reply scanomatic/io/pickler.py:48]: Good thing you have a decent store of bubbly beverages for all these upcoming moments of celebration.

**Pull request merged**